packages/reference/src/entries/WrappedEntryCard/FetchingWrappedEntryCard.tsx

```diff
@@ -13,6 +15,22 @@ export type EntryCardReferenceEditorProps = ReferenceEditorProps & {
   isDisabled: boolean;
   onRemove: () => void;
   cardDragHandle?: React.ReactElement;
+  renderCard?: Function; // TODO: What's the correct type to use here?
+};
+
+export type WrappedEntryCardProps = {
+  getAsset: (assetId: string) => Promise<unknown>;
+  getEntityScheduledActions: SpaceAPI['getEntityScheduledActions'];
+  getEntryUrl?: (entryId: string) => string;
+  isDisabled: boolean;
+  size: 'default' | 'small';
+  localeCode: string;
+  defaultLocaleCode: string;
+  allContentTypes: ContentType[];
+  entry: 'failed' | undefined | Entry; // TODO: Work out correct type for entry
```

Comment:
This should never be `'failed'`, making this a public interface would be quite ugly design plus, it seems not even necessary in `WrappedEntryCard`. There is some `entry?.sys.id` usage so maybe there's a valid case for `undefined`, although `FetchingWrappedEntryCard` seems to catch this case for us, so not sure if there's a point at all.